conference/submissions.md

```diff
@@ -47,22 +47,19 @@ at this stage ([COVID-19 decision timeline](/conference/timeline/)).
 
 ### Proposal types
 
-We welcome any type of proposal for this conference but our forms will be structured
-for five categories (see below). This is the
+This is the
 first Nordic-RSE meeting, so we especially welcome a variety of event
 types which promote engagement and community building, and are target
 to people who don't already identify as RSEs.
 
-Our venue is the KTH Student Union ([more details about the venue](/conference/practical/#venue)),
-and will have lots of places for
-informal discussions or close interactions.  But not *too* close,
-since COVID-19 isn't going away.
-
-We aim for recording and streaming, but the process is not yet decided
-and this is not required if you don't feel comfortable or it isn't
-suitable.  However, we are still aiming for an in-person event, since
-online can't serve the same purpose of building a new community (but
-would be glad to have hybrid online/in person events proposed).
+Please don't feel limited to classic lectures - we will consider any
+kind of event, such as discussions, networking sessions, hackathons,
+workshops, art exhibits, games, contests, etc.  Our primary categories
+to fill, though, are **plenary**, **talk/workshop**, **lightning
+talk**, **poster or exhibit**.  Plenaries are 30-45 minutes,
+talks/workshops sessions are up to 1.5 hours in a session (but could
```

Comment:
Somewhere else I said that a typical talk was 20 minutes.  Should we say what the typical talk length should be, or leave it vague?

Reply:
I would propose not using the suggestion I gave above, merging this right away, and new PR adjusts text.  I can do the folllow-up.

Reply:
OK I will merge and check the talk length. I think there may be some inconsistencies for historical reasons.